site/content/docs/0.0/utilities/borders.md

```diff
@@ -8,4 +8,293 @@ aliases:
+
+{{< example >}}
+{{< placeholder width="75" height="75" class="rounded" title="Example rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-top" title="Example top rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-end" title="Example right rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-bottom" title="Example bottom rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start" title="Example left rounded image" color="#999" >}}
+{{< /example >}}
+
+### Sizes
+
+Use the scaling classes for larger or smaller rounded corners. Sizes are `none`, `default`, `short`, `medium`, `tall`, `circle` and `pill`, and can be configured by modifying the utilities API.
+
+{{< example class="bd-example-rounded-utils" >}}
+{{< placeholder width="75" height="75" class="rounded-none" title="Example non-rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-default" title="Example default rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-short" title="Example short rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-medium" title="Example medium rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-tall" title="Example tall rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-circle" title="Completely round image" color="#999" >}}
+{{< placeholder width="150" height="75" class="rounded-pill" title="Rounded pill image" color="#999" >}}
+{{< /example >}}
+
+{{< example class="bd-example-rounded-utils" >}}
+{{< placeholder width="75" height="75" class="rounded-bottom-short" title="Example small rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start-medium" title="Example default left rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-end-circle" title="Example right completely round image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start-pill" title="Example left rounded pill image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-tall rounded-top-none" title="Example extra large bottom rounded image" color="#999" >}}
+{{< /example >}}
+
+{{< bootstrap-compatibility >}}
+
+Use the scaling classes for larger or smaller rounded corners. Sizes range from `0` to `5`, and can be configured by modifying the utilities API.
+
+{{< example class="bd-example-rounded-utils" >}}
+{{< placeholder width="75" height="75" class="rounded-0" title="Example non-rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-1" title="Example small rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-2" title="Example default rounded image" color="#999" >}}
```

Comment:
this utility is a bit weird, the map is like that :
```scss
        1: var(--#{$prefix}border-radius-sm),
        2: var(--#{$prefix}border-radius),
        3: var(--#{$prefix}border-radius-lg),
```
In OUDS, the variable "border-radius" is equal to the token "default", valued to 0. But "radius sm" is mapped to "radius-short". So we ended up with `rounded-2` that doesn't have any border radius but `rounded-1` has.
We probably need to change the mapping in utilities .

Reply:
This is a bit weird, but I'm wondering if we should keep the CSS variables for the Bootstrap compatibility only or not. If yes, maybe it's not that weird to have this kind of thing.

Reply:
We still have the problem in compatibility mode, the rounded-2 is valued to 0, but rounded-1 have a non zero value. We could modify the map in utilities, in order to remove the usage of `var(--#{$prefix}border-radius)` for the size 2 or we could change the value of this CSS variables in root for another value than the default one.

I find it more logical to keep the default value in the CSS variable and to change the variables used in the map. But in every case we could have some issue with other libs using those variables.

Reply:
We decided to keep the default token for the `rounded` helper, meaning it does nothing in Orange. We added a info callout to explain why. The mapping for `rounded-*` size in compatibility mode have been changed in order to remove `$border-radius` from this list for size 2. We introduced a arbitrary new value, without any variable for the size 5 (should be equivalent to an hypothetical xxxl)